### PR TITLE
Storybook practice

### PR DESCRIPTION
**Pull request by hikariumeno**
storybookを導入します

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,9 @@
     "lint:js": "eslint --ext .js,.vue --ignore-path .gitignore .",
     "lint:style": "stylelint **/*.{vue,css} --ignore-path .gitignore",
     "lint": "npm run lint:js && npm run lint:style",
-    "test": "jest"
+    "test": "jest",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
```
Comment (9renpoto) [inline package.json:15]: https://github.com/hikariumeno/nuxt.js-practice/blob/eb9c4bd6169e78601a786007571fb6d6687b2a49/.github/workflows/node.js.yml#L28

please insert `yarn build-storybook` before `test` after `npm run build` :)

Comment (9renpoto) [reply package.json:15]: oops, `npm run build-storybook`

File: .github/workflows/node.js.yml [hunk 2]
```diff
@@ -26,4 +26,5 @@ jobs:
         node-version: ${{ matrix.node-version }}
     - run: npm ci
     - run: npm run build --if-present
+    - run: npm run build-storybook
```
Comment (9renpoto) [inline .github/workflows/node.js.yml:29]: https://github.com/hikariumeno/nuxt.js-practice/pull/3/files#r499219364

OK, thank you :)

File: package.json (hunk 1 above)
Comment (hikariumeno) [reply package.json:15]: thank you, I fixed in this commit d02052ccc1f9ed896b97150e26bcdc969979776c

**Review by 9renpoto — APPROVED** [review]
lgtm

**Pull request merged**